src/lv_objx/lv_spinbox.c

```diff
@@ -289,7 +316,11 @@ void lv_spinbox_increment(lv_obj_t * spinbox)
         ext->value += ext->step;
 
     } else {
-        ext->value = ext->range_max;
+		// Rollover?
+		if (ext->rollover)
+			ext->value = ext->range_min + ext->step - 1;
```

Comment:
I haven't tested it but it seems incorrect at first look. Let say the range is 0..1000 and the step is 10.  In the case of rollover, the value will change from 1000 to 9. I'd expect to change the value to `ext->range_min`. What do you think?

Reply:
Well, I think it's just a matter of preference of what to do when reaching the limits. 
1. Behaving like that formula anyway
2. Stopping at maximum first the new value is more than max, then restating from the minimum (and vice versa)
3. Crossing the maximum anyway, but re-starting at minimum value (and vice versa)

Maybe the second is the best solution.

Reply:
> Maybe the second is the best solution.

I agree. Just to be sure, for example with 0..1000 range and step = 10 it would look like: 988, 998, 1000, 0, 10, 20. Did you mean the same?

Reply:
Yes, exactly.  I'll be reviewing the code ASAP.